_posts/2015-09-15-meetup.md

```diff
@@ -0,0 +1,57 @@
+---
+title: All-React Extravaganza
+author: aaron
+layout: post
+categories:
+- posts
+- meetups
+when: 2015-09-15T19:30:00-05:00
+---
+
+Webster's Dictionary [defines "extravaganza" as][extravaganza]:
+
+> a very large and exciting show or event
+
+And nothing says exciting like JavaScript frameworksssssss!
+
+<div class="sponsor-logo">
+  <img src="https://cldup.com/D3VCZfiOrY.gif" style="width: 300px; max-width: 100%;" alt="WHOA REACT JS" />
```

Comment:
whoa good point. would one of these work? we need something safe, but i'd still like it to be as offensive to the eyes as possible:

hypno:

![hypno](https://cldup.com/SIbzlox5dC.gif)

miami night club:

![night club](https://cldup.com/G5ajKtTSq6.gif)

drunk goggles:

![drunk goggles](https://cldup.com/L-nxALNYAl.gif)